### DISCUSSION TITLE
Support for Bukkit Plugins?

### DISCUSSION THREAD

**Discussion by Coderdude112** [itzg/docker-minecraft-server#1369] [Q&A] [answered]
Hello!

I am wanting to use plugins on my server. I have found the SPIGET_RESOURCES env variable but am unsure of how to add bukkit based plugins such as [this one](https://dev.bukkit.org/projects/openinv).

Am I mistaken in thinking this project has support for these plugins?

**Comment by itzg** [comment, marked as answer]
Rather than use the spiget approach, I would recommend using `MODS` described in

https://github.com/itzg/docker-minecraft-server#downloadable-modplugin-pack-for-forge-fabric-and-bukkit-like-servers

**Reply by Coderdude112** [reply]
Thanks!
